examples/stable/kafka/kafka-blueprint.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: kafka-connect-blueprint
+  namespace: kasten-io
+actions:
+  backup:
+    type: Deployment
+    phases:
+    - func: KubeTask
+      name: setupPhase
+      args:
+        namespace: kafka
+        podOverride:
+          containers:
+          - name: container
+            imagePullPolicy: IfNotPresent
+            livenessProbe:
+              httpGet:
+                path: /connectors
+                port: 8083
+              initialDelaySeconds: 60
+              periodSeconds: 5
```

Comment:
why is it required?

Reply:
If the connector doesn't start for any reason, or fails later on, the backup process will stop.

Reply:
sorry I didnt understand. how, adding `livenessProbe` is going to help us here?

Reply:
Yes, I agree, have removed the section from the blueprint. The scenario of connector failing at any staged has been handled inside the blueprint itself.